kubebench/base/kustomization.yaml

```diff
@@ -25,3 +25,13 @@ images:
   - name: gcr.io/kubeflow-images-public/kubebench/kubebench-example-tf-cnn-post-processor
     newName: gcr.io/kubeflow-images-public/kubebench/kubebench-example-tf-cnn-post-processor
     newTag: v0.4.0-13-g262c593
+vars:
```

Comment:
Nice, could you also update jupyter/jupyter/base/kustomization.yaml, jupyter-web-app/base/kustomization.yaml? They reference the namespace in the configmap, we might as well be consistent.

```
- name: namespace
  objref:
    kind: ConfigMap
    name: parameters
    apiVersion: v1
  fieldref:
    fieldpath: data.namespace
```

Reply:
good catch! thanks